### DISCUSSION TITLE
Could I use AIO in http?

### DISCUSSION THREAD

**Discussion by Pyloons** [nextcloud/all-in-one#639] [Questions] [answered]
I installed aio in my home local area network.but now it give me two choice: a new instance or restore a instance.I want to a new one but I needn't a domain.
I have changed the host machine's /etc/hostname for a domain, and use adguardhome's dns rewrite, and set the ufw of host machine allow http, https, 8080, 8443, 3478(tcp & udp).but aio just say no.
Could I use aio in http?

**Comment by szaimen** [comment, marked as answer]
Hello, using a local domain, ip-address or http for Nextcloud is not supported. But you may be able to configure that manually by following https://github.com/nextcloud/all-in-one/tree/main/manual-install#manual-installation

**Comment by lumpy64** [comment]
> I installed aio in my home local area network.but now it give me two choice: a new instance or restore a instance.I want to a new one but I needn't a domain. I have changed the host machine's /etc/hostname for a domain, and use adguardhome's dns rewrite, and set the ufw of host machine allow http, https, 8080, 8443, 3478(tcp & udp).but aio just say no. Could I use aio in http?

hey there! did this solution work for you? I'm encountering the same issue! thank you
